tests/acceptance/users.feature

```diff
@@ -13,76 +13,84 @@ Feature: users
 
   Scenario: Create super admin and login into the backend
     Given There is a add user link
-    When  I create a super admin with fields Name "prital", Login Name "prital", Password "prital", and Email "prital@gmail.com"
-    And I set assigned user group as an Administrator
-    Then I Save the  user
-    And Login in backend with username "prital" and password "prital"
+    And  I fill a super admin with fields Name "prital", Login Name "prital", Password "prital", and Email "prital@gmail.com"
+    When I set assigned user group as an Administrator
+    And I Save the  user
+    Then Login in backend with username "prital" and password "prital"
 
-  Scenario:create User without username fails
+  Scenario: Create User without username fails
     Given There is a add user link
     When I don't fill Login Name but fulfill remaining mandatory fields: Name "piyu", Password "piyu" and Email "piyu@gmail.com"
-    Then I Save the  user
-    And I see the "Invalid field:  Login Name" alert error
+    And I Save the  user
+    Then I see the title "Users: New"
+    But I see the alert error "Invalid field:  Login Name"
 
   Scenario: Create group
     Given There is a add new group link
     When I fill Group Title as a "Gsoc"
     And I save the Group
-    Then I should see the "Group successfully saved." message
+    Then I should wait for "Users: Groups" title and see the message "Group successfully saved."
 
   Scenario: Edit group
     Given I search and select the Group with name "Gsoc"
     And I set group Title as a "Gsoc_admin"
     When I save the Group
-    Then I should see the "Group successfully saved." message
+    Then I should wait for "Users: Groups" title and see the message "Group successfully saved."
 
   Scenario: Delete Group
     Given I search and select the Group with name "Gsoc_admin"
     When I Delete the Group "Gsoc_admin"
-    Then I confirm the group should have been deleted by getting the message "1 User Group successfully deleted."
+    Then I confirm the group should have been deleted by getting the title "Users: Groups" and see the message "1 User Group successfully deleted."
 
   Scenario: Create ACL level
     Given There is a add viewing access level link
     When I fill Level Title as a "joomla" and set Access as a public
     And I save the Access Level
-    Then I should be see the "Access level successfully saved." message
+    Then I wait for the title "Users: Viewing Access Levels" see the message "Access level successfully saved."
 
-  Scenario:  Edit ACL
+  Scenario: Edit ACL
     Given I search and select the Access Level with name "joomla"
     And I set Access Level title as a "Gsoc_joomla"
     When I save Access Level
-    Then I should be see the "Access level successfully saved." message
+    Then I wait for the title "Users: Viewing Access Levels" and see the message "Access level successfully saved."
```

Comment:
based in the comment https://github.com/joomla-projects/gsoc16_browser-automated-tests/pull/33/files#r68762732 this could be:

```gherkin
Then I wait for the title "Users: Viewing Access Levels"
And I see the message "Acces level succesfully saved."
```